mods/boats/init.lua

```diff
@@ -233,9 +233,11 @@ minetest.register_craftitem("boats:boat", {
 		end
 		pointed_thing.under.y = pointed_thing.under.y + 0.5
 		boat = minetest.add_entity(pointed_thing.under, "boats:boat")
```

Comment:
pretty sure there should be a `local` here? Otherwise it overrides the global boat API with an entity reference